mungegithub/issue-labeler/simple_app.py

```diff
@@ -0,0 +1,125 @@
+import os
+import simplejson
+import logging
+from logging.handlers import RotatingFileHandler
+
+import numpy as np
+from flask import Flask, request
+from sklearn.feature_extraction import FeatureHasher
+from sklearn.externals import joblib
+from sklearn.linear_model import SGDClassifier
+from nltk.tokenize import RegexpTokenizer
+from nltk.stem.porter import PorterStemmer
+app = Flask(__name__)
+
+#Parameters
+team_fn= "./models/trained_teams_model.pkl"
+component_fn= "./models/trained_components_model.pkl"
+logFile = "/tmp/issue-labeler.log"
+logSize = 1024*1024*100
+numFeatures = 262144
+myLoss = 'hinge'
+myAlpha = .1
+myPenalty = 'l2'
+myHasher = FeatureHasher(input_type="string", n_features= numFeatures, non_negative=True)
+myStemmer = PorterStemmer()
+tokenizer = RegexpTokenizer(r'\w+')
+
+
+try:
+  stop_words = []
+  if not stopwords:
+    stop_fn = "./data/stopwords.txt"
+  with open(stop_fn, 'r') as f:
+    stopwords = [word.strip() for word in f]
+except:
+  #don't remove any stopwords
+  stopwords = []
+
+@app.errorhandler(500)
+def internal_error(exception):
+  app.logger.error(exception)
+  return "Internal Server Error-> Check the Logs"
```

Comment:
I would have done: `return str(exception)`. We already know it's an internal error (because of 500). And it's much simpler than checking the logs.